modules/rostests/winetests/comctl32/pager.c

```diff
@@ -40,8 +40,13 @@ static const WCHAR large_w[] =
     L"You should have received a copy of the GNU Lesser General Public License along with this ...";
 static const WCHAR large_truncated_65_w[65] =
     L"You should have received a copy of the GNU Lesser General Public";
+#ifdef __REACTOS__
```

Comment:
Please copy-paste into the description a screenshot or a text excerpt of the warnings that will get fixed.

Reply:
Okay, I attached it myself.